velox/functions/sparksql/String.h

```diff
@@ -762,7 +763,63 @@ struct LeftFunction {
     auto byteRange =
         stringCore::getByteRange<isAscii>(input.data(), start, length);
 
-    // Generating output string
+    // Generating output string.
+    result.setNoCopy(StringView(
+        input.data() + byteRange.first, byteRange.second - byteRange.first));
+  }
+};
+
+/// right function
+/// right(string, length) -> string
+/// Returns the rightmost length characters from the string.
+/// Returns an empty string if length is less or equal than 0.
+/// Returns the original string if length is larger than the string length.
+template <typename T>
+struct RightFunction {
```

Comment:
I just noticed Spark maintained a replacement for the Right function. In this case, probably it is converted as substring which is already supported. Could you help confirm? Thanks.
https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala#L2008-L2016